modules/claims_api/config/routes.rb

```diff
@@ -2,21 +2,21 @@
 
 ClaimsApi::Engine.routes.draw do
   match '/metadata', to: 'metadata#index', via: [:get]
-  match '/v0/healthcheck', to: 'metadata#healthcheck', via: [:get]
-  match '/v1/healthcheck', to: 'metadata#healthcheck', via: [:get]
-  match '/v0/upstream_healthcheck', to: 'metadata#upstream_healthcheck', via: [:get]
-  match '/v1/upstream_healthcheck', to: 'metadata#upstream_healthcheck', via: [:get]
+  match '/v0/upstream_healthcheck', to: 'upstream_healthcheck#index', via: [:get], defaults: { format: 'json' }
+  match '/v1/upstream_healthcheck', to: 'upstream_healthcheck#index', via: [:get], defaults: { format: 'json' }
```

Comment:
https://github.com/department-of-veterans-affairs/vets-api/pull/5718/files#r560555774